src/main/java/com/celements/payment/container/EncryptedComputopData.java

```diff
@@ -0,0 +1,35 @@
+package com.celements.payment.container;
+
+import javax.annotation.concurrent.Immutable;
+import javax.validation.constraints.NotNull;
+
+import com.google.common.base.Preconditions;
+
+@Immutable
+public class EncryptedComputopData {
+
+  private final String cipherText;
+  private final int plainDataLength;
+
+  public EncryptedComputopData(@NotNull String cipherText, int plainDataLength) {
+    Preconditions.checkNotNull(cipherText);
+    this.cipherText = cipherText;
+    this.plainDataLength = plainDataLength;
+  }
+
+  public @NotNull String getCipherText() {
```

Comment:
i've seen `new EncryptedComputopData("", ...)`, consider a Nullable cipherText with `emptyToNull` and `public Optional<String> getCipherText()` to make clear whether we have no encrypted data